src/pages/ProductDetailPage.js

```diff
@@ -0,0 +1,22 @@
+import './ProductDetailPage.css';
+import {
+  Header,
+  Footer,
+  SingleProductCard,
+  ProductDescription,
+} from '../components';
+import { Fragment } from 'react';
+
+const ProductDetailPage = props => {
+  return (
+    <Fragment>
+      <Header />
```

Comment:
add the header in the routes component that way its going to be rendered on every page